.github/workflows/gql-perf-arch-github_workflow.yml

```diff
@@ -0,0 +1,70 @@
+name: gql-perf-arch-github_workflow
+on:
+  push:
+    branches:
+      - master
+  workflow_dispatch:
+    inputs:
+      COMMIT_HASH_INPUT:
+        description: 'Commit hash'
+        required: true
+      CLASSES_TO_EXECUTE_INPUT:
+        description: 'Classes to test'
+        required: false
+      PULL_REQUEST_NUMBER_INPUT:
+        description: 'Pull request number'
+        required: false
+env:
+  #Actions secrets (keys)
+  PROJECT_ID: ${{ secrets.PROJECT_ID }}
+  QUEUE_ID: ${{ secrets.QUEUE_ID }}
+  LOCATION: ${{ secrets.LOCATION }}
+  WORKFLOW_URL: ${{ secrets.WORKFLOW_URL }}
+  SERVICE_ACCOUNT_EMAIL: ${{ secrets.SERVICE_ACCOUNT_EMAIL }}
```

Comment:
why is this needed?

Reply:
I already updated the description of the draft PR to explain why we need each of those variables.